packages/fxa-react/styles/tooltips.css

```diff
@@ -2,14 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
I think if we're going to have two different versions of this, the other still living in content-server (which is okay by me since content-server's has `.tooltip-container` as well as `.password-strength`), we may want this one to stay in Settings. Alternatively, we can just remove `.input-balloon` styles from content-server's `tooltips.css` so that `input-balloon` is the class shared between the two. Otherwise, I'm pretty sure content-server is going to get `input-balloon` twice since it pulls in this file from `fxa-react` which might lead to some confusion later if we modify either file and something's not updating that we think should.

BTW I'd probably suggest moving all of these TW class names to the TSX directly on `PasswordStrengthBalloon` instead of using a component class but I'm pretty sure we're going to use this same class when we work on the 2nd input balloon on signup, so this makes sense 👍

Reply:
That all makes sense. I'd initially removed the styles from content-server and saw some failing tests, but I think that's because I'd moved the styles to fxa-settings instead of fxa-react (oops).